### ISSUE TITLE
Ckeditor5-vue Table not showing ballon or table properties

### ISSUE THREAD

**Issue by thamibn** [ckeditor/ckeditor5-vue#187]
Hi,

I have setup the ckeditor using npm it is showing but i when i insert a table it does not show the table properties or ballon like on demo.

![Screenshot from 2021-06-10 23-44-42](https://user-images.githubusercontent.com/28577943/121601394-2d368000-ca46-11eb-82b5-ee2505ee45c7.png)

```
editor: DecoupledEditor,
            editorData: tenderDoc,
            editorConfig: {
            }
```

thaats how my setup looks.

**Comment by FilipTokarski** [comment]
Hi, to see table toolbar you need to add `tableToolbar` plugin ( if you don't have it already ) and its options to the editor config, like in [this example](https://ckeditor.com/docs/ckeditor5/latest/features/table.html#installation). Apart from that, please see the docs on [editor configuration](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/configuration.html) and [integrating editor with Vue](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/vuejs-v3.html).

**Comment by thamibn** [comment]
Hi @FilipTokarski;

I have tried that just now and i am getting the below error.
![Screenshot from 2021-06-14 11-19-16](https://user-images.githubusercontent.com/28577943/121869714-9cd89380-cd02-11eb-9401-36b0654a91ed.png)


i installed the build not the src, it supposed to work out of the box isnt?

my setup.
```
1. import CKEditor from '@ckeditor/ckeditor5-vue2';
    Vue.use( CKEditor );
    
2. import DecoupledEditor from '@ckeditor/ckeditor5-build-decoupled-document';
    import Table from '@ckeditor/ckeditor5-table/src/table';
    import TableToolbar from '@ckeditor/ckeditor5-table/src/tabletoolbar';
    
3. editor: DecoupledEditor,
    editorData: tenderDoc,
    editorConfig: {
        plugins: [ Table, TableToolbar],
        table: {
            contentToolbar: [ 'tableColumn', 'tableRow', 'mergeTableCells' ]
        }
    }
    
and the above setup results to the above error.
```

**Comment by FilipTokarski** [comment]
You are confusing [editor build](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/overview.html) with [creating editor from source](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/overview.html). Adding plugins to the build like this is not allowed, hence the [duplicated-modules-error](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/support/error-codes.html#error-ckeditor-duplicated-modules). In this case you should only add table options to the config, because the `Table` package is already in the decoupled document build.
Please make yourself familiar with the documentation and guides before submitting issues on GitHub.

**Comment by thamibn** [comment]
@FilipTokarski just edited the config like below.
```
editorConfig: {
    table: {
        contentToolbar: [
            'tableColumn',
            'tableRow',
            'mergeTableCells',
            'tableCellProperties',
            'tableProperties'
        ]
    }
  }
```

now there is the below warnings
![Screenshot from 2021-06-14 11-48-10](https://user-images.githubusercontent.com/28577943/121873262-7d436a00-cd06-11eb-9df1-403f3fdb5974.png)

I have read the docs thats why i told you i have the build not from src because of your first answer.
How can i solve the above warning, since build comes with all the plugins.

**Comment by FilipTokarski** [comment]
[`TableProperties` and `TableCellProperties` plugins](https://ckeditor.com/docs/ckeditor5/latest/features/table.html#table-and-cell-styling-tools-2) are not part of the decoupled editor build ( you can [see the default configuration here](https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-build-decoupled-document/src/ckeditor.js) ). Now, if you want to add them ( or any other additional plugins ), you need to [build the editor from source](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/vuejs-v3.html#using-ckeditor-from-source). But if you just want to remove the warnings - remove `tableProperties` and `tableCellProperties` from the configuration.

**Comment by thamibn** [comment]
@FilipTokarski Thanks for the info above, I have removed accordingly as per your advice still the pop up does not come up when i click a cell?
![Screenshot from 2021-06-14 12-46-31](https://user-images.githubusercontent.com/28577943/121880686-96e8af80-cd0e-11eb-8d35-b8beb711320a.png)


now my editorConfig just look like this
```
editorConfig: {
                table: {
                    contentToolbar: [
                        'tableColumn',
                        'tableRow',
                        'mergeTableCells',
                    ]
                }
            }
```

**Comment by zomby22** [comment]
make sure you also include this:

```
.then( editor => {
				window.editor = editor;

				const tableConfig = {
					toolbar: [ 'tableRow', 'tableColumn', 'mergeTableCells', 'tableProperties', 'tableCellProperties', 'toggleTableCaption' ]
				};
```

**Comment by pomek** [comment]
@thamibn, did you manage to fix the problem?

**Comment by thamibn** [comment]
> @thamibn, did you manage to fix the problem?

@pomek Yes, the issue was that the CKeditor was on a modal so they were conflicting probably z-index...

So I removed Ckeditor from bootstrap modal to a stand alone page.

**Comment by thamibn** [comment]
Satisfied with the workaround.

**Comment by VinoyPX** [comment]
Works on bootstrap modal by changing the z-index of .ck-balloon-panel
.ck-balloon-panel {
    z-index: value_greter_than_model
}

**Comment by Witoso** [comment]
Some examples for a few CSS frameworks can be found in [our documentation](https://ckeditor.com/docs/ckeditor5/latest/installation/integrations/css.html).
